gpkit/constraints/bounded.py

```diff
@@ -74,27 +74,32 @@ def __init__(self, constraints, substitutions=None, verbosity=1,
                                      if vk not in constraints.substitutions)
         bounding_constraints = varkey_bounds(self.bounded_varkeys,
                                              self.lowerbound, self.upperbound)
-        super(Bounded, self).__init__([constraints, bounding_constraints],
-                                      substitutions)
+        super(Bounded, self).__init__([constraints, bounding_constraints])
 
     def sens_from_dual(self, las, nus):
         "Return sensitivities while capturing the relevant lambdas"
         n = sum(map(bool, [self.lowerbound, self.upperbound]))
         self.bound_las = las[-n*len(self.bounded_varkeys):]
         return super(Bounded, self).sens_from_dual(las, nus)
 
+    def as_gpconstr(self, x0, substitutions=None):
+        gp_constrs = ConstraintSet.as_gpconstr(self, x0, substitutions)
+        return Bounded(gp_constrs, self.verbosity,
+                       lower=self.lowerbound, upper=self.upperbound)
+
     def process_result(self, result):
         "Creates (and potentially prints) a dictionary of unbounded variables."
-        lam = self.bound_las
+        if not self.bound_las:
+            return  # must be an SP Bounded, boundedness was solved in the GP
```

Comment:
Not so much there's nothing to do, but there's nothing that needs doing; it's already been done at the GP level.